helpers/disk.py

```diff
@@ -18,13 +18,14 @@
 from ovs.dal.lists.diskpartitionlist import DiskPartitionList
 from ..helpers.storagerouter import StoragerouterHelper
 
-
 class DiskHelper(object):
     """
     DiskHelper class
     """
 
     def __init__(self):
+        from ..helpers.storagerouter import StoragerouterHelper
```

Comment:
Same import as on the top of the file, not really sure why
Doesn't seem to be used either